index.html

```diff
@@ -47,15 +47,25 @@
       <header class="header">
         <a class="owner">John Doe</a>
         <nav class="menu-button"><i class="fa fa-bars"></i></nav>
+        <div class="menu">
+          <ul>
+              <li><a href="#">Portfolio</a></li>
+              <li><a href="#">About</a></li>
+              <li><a href="#">Contact</a></li>
+              <li><a href="#"><i class="fas fa-envelope"></i></a></li>
+          </ul>
+        </div>
```

Comment:
- [ ] Kindly reduce the height of the header to  look more like the Figma design, please check the screenshot below ⬇️